### PR TITLE
Redesigned UTXO vs Token FAQ

### LINKED ISSUE #1019: `token vs utxo` - update UI to match new design language
#### What UI/UX improvements would you like to be implemented: 
- Update UI of DFI Token vs UTXO screen found in Convert Screen

#### Why is this needed:
- To be consistent with overall app design

#### Which component/screens does this concern?:
- DFI Token vs UTXO screen

image

### PR DESCRIPTION
**Pull request by fullstackninja864**
#### What kind of PR is this?:

/kind feature

#### Which issue(s) does this PR fixes?:

Fixes #1019

#### Developer Checklist:

- [x] Read your code changes at least once
- [x] Tested on iOS/Android device (e.g, No crashes, library supported etc.)
- [x] No console errors on web
- [x] Tested on Light mode and Dark mode*
- [x] Your UI implementation visually matched the rendered design*
- [x] Unit tests*
- [ ] Added e2e tests*
- [ ] Added translations*

### CODE REVIEW

File: mobile-app/app/screens/AppNavigator/screens/Balances/screens/TokensVsUtxoScreen.tsx [hunk 1]
```diff
@@ -8,86 +8,110 @@ import { translate } from '@translations'
...
-        dark={tailwind('text-darksuccess-500')}
+        size={16}
+        name='check-circle'
         iconType='MaterialIcons'
-        light={tailwind('text-success-500')}
-        name='check'
-        size={24}
-        style={tailwind('mr-2')}
+        light={tailwind('text-green-500')}
+        dark={tailwind('text-green-500')}
```
Comment (kyleleow) [inline mobile-app/app/screens/AppNavigator/screens/Balances/screens/TokensVsUtxoScreen.tsx:103]:
```suggestion
        light={tailwind('text-success-500')}
        dark={tailwind('text-darksuccess-500')}
```

File: mobile-app/app/screens/AppNavigator/screens/Balances/screens/TokensVsUtxoScreen.tsx [hunk 2]
```diff
@@ -8,86 +8,110 @@ import { translate } from '@translations'
...
   return (
-    <View style={tailwind('flex flex-row items-center pb-5')}>
-      <TokenIcon style={tailwind('mr-2')} />
-
-      <ThemedText style={tailwind('text-lg')}>
-        {translate('screens/TokensVsUtxoScreen', label)}
-      </ThemedText>
+    <View style={tailwind('mb-6')}>
+      <View style={tailwind('flex-row items-center flex-grow mb-2')}>
+        <TokenIcon width={20} height={20} />
```
Comment (kyleleow) [inline mobile-app/app/screens/AppNavigator/screens/Balances/screens/TokensVsUtxoScreen.tsx:62]:
```suggestion
        <TokenIcon width={24} height={24} />
```

**Review by kyleleow — CHANGES_REQUESTED** [review]

File: mobile-app/app/screens/AppNavigator/screens/Balances/screens/TokensVsUtxoScreen.tsx [hunk 3]
```diff
@@ -8,86 +8,110 @@ import { translate } from '@translations'
...
-          {translate('screens/TokensVsUtxoScreen', 'UTXO DFI is the main form DFI and is used for core cryptocurrency purposes like send, receive and fees. DFI tokens are used for DeFi functions such as swaps and liquidity mining.')}
-        </ThemedText>
-
-        <ThemedText style={tailwind('pb-4')}>
-          {translate('screens/TokensVsUtxoScreen', 'Your DFI balance includes both UTXO and tokens.')}
+        <ThemedText
+          light={tailwind('text-gray-700')}
+          dark={tailwind('text-gray-200')}
+        >
+          {translate('screens/TokensVsUtxoScreen', 'DFI exists in two forms - UTXO and Token They can be converted with one another to serve different use cases within your light wallet.')}
```
Comment (thedoublejay) [inline mobile-app/app/screens/AppNavigator/screens/Balances/screens/TokensVsUtxoScreen.tsx:23]:
```suggestion
          {translate('screens/TokensVsUtxoScreen', 'DFI exists in two forms - UTXO and Token. They can be converted with one another to serve different use cases within your light wallet.')}
```

File: shared/translations/languages/de.json [hunk 4]
```diff
@@ -293,16 +293,19 @@
...
-    "Your DFI balance includes both UTXO and tokens.": "Dein DFI-Guthaben umfasst sowohl UTXO als auch Token.",
-    "Core crypto services": "Zentrale Kryptodienste",
-    "Send to other wallets": "An andere Wallets senden",
-    "Receive tokens": "Token empfangen",
-    "Service fees": "Servicegebühr",
-    "DeFi services": "DeFi-Dienste",
-    "Liquidity mining": "Liquiditätsmining",
-    "Atomic Swaps": "Atomic Swaps"
+    "Learn about UTXOs and Tokens": "Erfahre mehr über UTXOs und Token",
+    "DFI exists in two forms - UTXO and Token They can be converted with one another to serve different use cases within your light wallet.": "DFI gibt es in zwei Formen: UTXO und Token. Diese können in beide Richtungen in einander umgewandelt werden, um verschiedene Verwendungszwecke in deiner Lightwallet zu erfüllen.",
```
Comment (thedoublejay) [inline shared/translations/languages/de.json:297]: Need to update the key also from the previous typo

**Review by thedoublejay — CHANGES_REQUESTED** [review]

**Review by thedoublejay — APPROVED** [review]

**Review by kyleleow — APPROVED** [review]

**Pull request merged**